docs/user/guides/setup/configuration-file.rst

```diff
@@ -0,0 +1,223 @@
+
+There are some parts of the templates that you can leave in place:
+
+Comments
+  We added a lot of comments that explain stuff. They begin with a ``#``.
+
+Commented out features
+  Similar to comments, we also use the ``#`` in front of some configuration options that may or may not be relevant.
+  To enable something that's commented out,
+  simply delete the ``#`` and a blank space in front of the YAML code that you want to enable.
+
+``version`` key
+  The version key tells the system how to read the rest of the configuration file.
+  The current and only supported version is **version 2**.
+
+
+Adjust: ``build.os``
+~~~~~~~~~~~~~~~~~~~~
+
+In our examples,
+we are using Read the Docs' custom image based on the latest Ubuntu LTS release.
+LTS means long-term support,
+meaning that your builds should not break within next many years.
+
+However,
+you should pay attention to this field if your current project needs to build on an older version of Ubuntu,
+or in the future when you need features from a newer Ubuntu.
+
+.. seealso::
+
+   :ref:`config-file/v2:build.os`
+     Configuration file reference with all values possible for ``build.os``.
+
+
+Adjust: Python configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using Python in your builds,
+you should define the Python version in ``build.tools.python``.
+
+The ``python`` key contains a list of sub-keys,
+specifying the requirements to install.
+
+- Use ``python.install.package`` to install the project itself as a Python package using pip
+- Use ``python.install.requirements`` to install packages from a requirements file
+- Use ``build.jobs`` to install packages using Poetry or PDM
+
+.. seealso::
+
+   :ref:`config-file/v2:build.tools.python`
+     Configuration file reference with all Python versions available for ``build.tools.python``.
+
+   :ref:`config-file/v2:python`
+     Configuration file reference for configuring the Python environment activated by ``build.tools.python``.
+
+Adjust: Sphinx and MkDocs version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are using either the ``sphinx`` or ``mkdocs`` builder,
+then Sphinx or MkDocs will be installed automatically in its latest version.
+
+But we recommend that you specify the version that your documentation project uses.
+The ``requirements`` key is a file path that points to a text (``.txt``) file
+that lists the Python packages you want Read the Docs to install.
+
+.. seealso::
+
+   :ref:`guides/reproducible-builds:Use a requirements file for Python dependencies`
+      This guide explains how to specify Python requirements,
+      such as the version of Sphinx or MkDocs.
+
+   :ref:`config-file/v2:sphinx`
+     Configuration file reference for configuring the Sphinx builder.
+
+   :ref:`config-file/v2:mkdocs`
+     Configuration file reference for configuring the MkDocs builder.
+
+.. _config_howto_build.tools:
+
+Adjust: Additional build environments?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
To keep nomenclature consistent, probably "build tools" here, even though that is a term specific to RTD

```suggestion
Adjust: Additional build tools
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```